SU-SWS/stanford_basic
# Update stanford_basic.info.yml

# READY FOR REVIEW

# Summary
- Makes this theme dependant on components module because it is.

# Review By (Date)
- Not urgent. Wait for Drupal 8.9

# Urgency
- Low, wait for Drupal 8.9

# Steps to Test

1. Check out this branch
2. Install install profile

# Affected Projects or Products
- Just this theme

# Associated Issues and/or People
- https://www.drupal.org/project/drupal/issues/474684

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
